src/test/java/seedu/address/logic/commands/GemCommandTest.java

```diff
@@ -23,18 +24,75 @@
 
 public class GemCommandTest {
 
-    public static final Path COMPILED_PATH_1 = Paths.get("src/test/data/GemCommandTest/compiledsem1GEmodules.json");
-    public static final Path COMPILED_PATH_2 = Paths.get("src/test/data/GemCommandTest/compiledsem2GEmodules.json");
+    public static final Path GEH_PATH_1 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem1GEHmodules.json");
+    public static final Path GET_PATH_1 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem1GETmodules.json");
+    public static final Path GES_PATH_1 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem1GESmodules.json");
+    public static final Path GEQ_PATH_1 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem1GEQmodules.json");
+    public static final Path GER_PATH_1 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem1GERmodules.json");
+
+    public static final Path GEH_PATH_2 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem2GEHmodules.json");
+    public static final Path GET_PATH_2 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem2GETmodules.json");
+    public static final Path GES_PATH_2 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem2GESmodules.json");
+    public static final Path GEQ_PATH_2 = Paths.get("src/test/data/GemCommandTest"
+        + "/sem2GEQmodules.json");
+    public static final Path GER_PATH_2 = Paths.get("src/test/data/GemCommandTest"
```

Comment:
Since GER and GEQ are the same both Sem 1 and 2, I think we can reuse the paths? HAHAH